### PR TITLE
Add nodeinfo 2.1 and 2.2

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: src/Module/NodeInfo122.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+use Friendica\Core\L10n;
+use Friendica\Model\Nodeinfo;
+use Friendica\Util\Profiler;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Version 2.2 of Nodeinfo, a standardized way of exposing metadata about a server running one of the distributed social networks.
+ * @see https://github.com/jhass/nodeinfo/blob/master/PROTOCOL.md
+ */
+class NodeInfo122 extends BaseModule
```
Comment (MrPetovan) [inline src/Module/NodeInfo122.php:37]: What's the purpose of the initial 1 in the class name?

Comment (annando) [reply src/Module/NodeInfo122.php:37]: See the class `NodeInfo210` for that. In that other class we have got version 1.0 of Nodeinfo 2.

Comment (MrPetovan) [reply src/Module/NodeInfo122.php:37]: Thanks, I was wondering if it was this kind of idiosyncrasy but the provided documentation about NodeInfo doesn't mention a second one with its own versioning.

File: src/Module/WellKnown/NodeInfo.php [hunk 2]
```diff
@@ -39,6 +39,10 @@ protected function rawContent(array $request = [])
 				 'href' => DI::baseUrl() . '/nodeinfo/1.0'],
 				['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.0',
 				 'href' => DI::baseUrl() . '/nodeinfo/2.0'],
+				 ['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.1',
+				 'href' => DI::baseUrl() . '/nodeinfo/2.1'],
+				 ['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.2',
```
Comment (MrPetovan) [inline src/Module/WellKnown/NodeInfo.php]: These lines don't match the original alignment, please fix either the first two or the last two to normalize the indentation.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/NodeInfo122.php (hunk 1 above)
Comment (annando) [reply src/Module/NodeInfo122.php:37]: Yeah, this is the spec: https://github.com/jaywink/nodeinfo2

I now updated the link to the documentation in the class.

File: src/Module/WellKnown/NodeInfo.php (hunk 2 above)
Comment (annando) [reply src/Module/WellKnown/NodeInfo.php]: It's now normalized.

**Comment by MrPetovan** [comment]
@nupplaphil Where is `check-php-cs.sh` defined?
```
+ /check-php-cs.sh

In ConfigurationResolver.php line [6](https://ci.friendi.ca/repos/2/pipeline/1714/14#L6)02:
                                                                          
  [PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]  
  For multiple paths config parameter is required.                        
                                                                          

Exception trace:
  at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php:602
 PhpCsFixer\Console\ConfigurationResolver->computeConfigFiles() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php:225
 PhpCsFixer\Console\ConfigurationResolver->getConfig() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php:646
 PhpCsFixer\Console\ConfigurationResolver->getFormat() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/ConfigurationResolver.php:4[7](https://ci.friendi.ca/repos/2/pipeline/1714/14#L7)6
 PhpCsFixer\Console\ConfigurationResolver->getReporter() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php:151
 PhpCsFixer\Console\Command\FixCommand->execute() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/symfony/console/Command/Command.php:256
 Symfony\Component\Console\Command\Command->run() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/symfony/console/Application.php:[9](https://ci.friendi.ca/repos/2/pipeline/1714/14#L9)71
 Symfony\Component\Console\Application->doRunCommand() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/symfony/console/Application.php:290
 Symfony\Component\Console\Application->doRun() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/src/Console/Application.php:90
 PhpCsFixer\Console\Application->doRun() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/symfony/console/Application.php:166
 Symfony\Component\Console\Application->run() at /woodpecker/src/github.com/friendica/friendica/bin/dev/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/php-cs-fixer:[11](https://ci.friendi.ca/repos/2/pipeline/1714/14#L11)5

fix [--path-mode PATH-MODE] [--allow-risky ALLOW-RISKY] [--config CONFIG] [--dry-run] [--rules RULES] [--using-cache USING-CACHE] [--cache-file CACHE-FILE] [--diff] [--diff-format DIFF-FORMAT] [--format FORMAT] [--stop-on-violation] [--show-progress SHOW-PROGRESS] [--] [<path>...]
```

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**